File: fibonacci.c
#include <stdio.h>

main()
{
	int fib[2] = {0, 1};
	int fibnum, fib_temp;

	printf("The first 10 Fibonacci numbers are:\n");
	printf("0\n");
	printf("1\n");

	for(fibnum = 3; fibnum < 11; fibnum++)
	{
		fib_temp = fib[0] + fib[1];
		fib[0] = fib[1];
		fib[1] = fib_temp;

		printf("%i\n", fib_temp);
	}

	return 0;
}

		

File: fibonacci.py
#///////////////////////////////////////////////////////////////
#
#	Dominic Spinosa
#	Application Security
#	Assignment 1 - Turing Complete Sandbox
#	Test Case 2
#	9/12/13
#
#	This program calculates and outputs the first ten 
#	numbers of the Fibonacci sequence.
#
#///////////////////////////////////////////////////////////////

fib = [0, 1]
fibnum, fib_temp

print("The first 10 Fibonacci numbers are: ")
print("0")
print("1")

for fibnum in range(3, 11):
	fib_temp = fib[0] + fib[1]
	fib[0] = fib[1]
	fib[1] = fib_temp

	print(fib_temp)

